script/1_0_2current_en_script/Arcueid/Day 2/02_00_ARC02_4B.txt

```diff
@@ -54,7 +54,7 @@ Although it's just a rice ball and miso soup."
 <Page15>
 "......!"
 What a thoughtful maid she is...!
-I'm sure she'll be scolded if Akiha finds out about the phone this morning, but to think that she would go this far...
+I'm sure she'll be scolded if Akiha finds out about that, or about the phone this morning, but to think that she would go this far...//　朝の携帯電話といい、秋葉に見つかったら叱られるのは彼女だろうに、ここまで気を遣ってくれるなんて……。
```

Comment:
If the first thing that can be found out about is the meal, I think that should be a `this`